aspnetcore/migration/mvc.md

```diff
@@ -38,29 +209,197 @@ Create a new *empty* ASP.NET Core web app with the same name as the previous pro
+
+* Add a *Controllers* folder.
+
+* Add a **Controller Class** named *HomeController.cs* to the *Controllers* folder.
+
+![Add New Item dialog](mvc/_static/add_mvc_ctl.png)
+
+* Add a *Views* folder.
+
+* Add a *Views/Home* folder.
+
+* Add a **Razor View** named *Index.cshtml* to the *Views/Home* folder.
+
+![Add New Item dialog](mvc/_static/view.png)
+
+The project structure is shown below:
+
+![Solution Explorer showing files and folders of WebApp1](mvc/_static/project-structure-controller-view.png)
+
+Replace the contents of the *Views/Home/Index.cshtml* file with the following markup:
+
+```html
+<h1>Hello world!</h1>
+```
+
+Run the app.
+
+![Web app open in Microsoft Edge](mvc/_static/hello-world.png)
+
+See [Controllers](xref:mvc/controllers/actions) and [Views](xref:mvc/views/overview) for more information.
+
+Now that we have a minimal working ASP.NET Core project, we can start migrating the following functionality from the ASP.NET MVC project:
 
-::: moniker range="< aspnetcore-2.0"
+* client-side content (CSS, fonts, and scripts)
 
-* When targeting .NET Core or .NET Framework, packages commonly used packages by MVC apps are listed individually in the project file.
+* controllers
+
+* views
+
+* models
+
+* bundling
+
+* filters
+
+* Log in/out, Identity (This is done in the next tutorial.)
+
+## Controllers and views
+
+* Copy each of the methods from the ASP.NET MVC `HomeController` to the new `HomeController`. Note that in ASP.NET MVC, the built-in template's controller action method return type is [ActionResult](https://msdn.microsoft.com/library/system.web.mvc.actionresult(v=vs.118).aspx); in ASP.NET Core MVC, the action methods return `IActionResult` instead. `ActionResult` implements `IActionResult`, so there's no need to change the return type of your action methods.
+
+* Copy the *About.cshtml*, *Contact.cshtml*, and *Index.cshtml* Razor view files from the ASP.NET MVC project to the ASP.NET Core project.
+
+* Run the ASP.NET Core app and test each method. We haven't migrated the layout file or styles yet, so the rendered views only contain the content in the view files. You won't have the layout file generated links for the `About` and `Contact` views, so you'll have to invoke them from the browser (replace **4492** with the port number used in your project).
+
+  * `http://localhost:4492/home/about`
+
+  * `http://localhost:4492/home/contact`
+
+![Contact page](mvc/_static/contact-page.png)
+
+Note the lack of styling and menu items. We'll fix that in the next section.
+
+## Static content
+
+In previous versions of ASP.NET MVC, static content was hosted from the root of the web project and was intermixed with server-side files. In ASP.NET Core, static content is hosted in the *wwwroot* folder. You'll want to copy the static content from your old ASP.NET MVC app to the *wwwroot* folder in your ASP.NET Core project. In this sample conversion:
```

Comment:
In ASP.NET MVC 5 and earlier, ...

Reply:
Copy the static content from the ASP.NET MVC app to the *wwwroot* folder in the ASP.NET Core project.

Much better without `you`.

Reply:
@scottaddie  recommend not using old ASP.NET MVC

* Just use ASP.NET MVC
* the existing ASP.NET MVC